src/containers/Procedures.js

```diff
@@ -1,26 +1,35 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { Header } from '../components/Header';
+import { Redirect } from "react-router-dom"
 import { menuStateChange, redirectFromPage } from '../action/HeaderActions';
 import './App.css'
 
 class Procedures extends Component {
+
     render() {
-        return (
-            <Header />
-        )
+        const isMenuActive = this.props.procedures.isUserMenuActivated,
+        isRedirected = this.props.procedures.isRedirected,
+        navBarActive = this.props.procedures.navBarActive;
```

Comment:
I can use https://medium.com/@stasonmars/%D0%B4%D0%B5%D1%81%D1%82%D1%80%D1%83%D0%BA%D1%82%D1%83%D1%80%D0%B8%D0%B7%D0%B0%D1%86%D0%B8%D1%8F-%D0%B2-es6-%D0%BF%D0%BE%D0%BB%D0%BD%D0%BE%D0%B5-%D1%80%D1%83%D0%BA%D0%BE%D0%B2%D0%BE%D0%B4%D1%81%D1%82%D0%B2%D0%BE-b865bb71f376 here:
const { isUserMenuActivated, isRedirected, navBarActive } = this.props.procedures;